packages/braintree-integration/src/braintree-fastlane/braintree-fastlane-payment-strategy.ts

```diff
@@ -19,10 +21,7 @@ import {
     PaymentRequestOptions,
     PaymentStrategy,
 } from '@bigcommerce/checkout-sdk/payment-integration-api';
-import {
-    isPayPalFastlaneCustomer,
-    PayPalCommerceInitializationData,
-} from '@bigcommerce/checkout-sdk/paypal-commerce-utils';
+import { isPayPalFastlaneCustomer } from '@bigcommerce/checkout-sdk/paypal-commerce-utils';
```

Comment:
It might be better to find another method somewhere in Braintree package

Reply:
fixed